.github/workflows/cmake.yml

```diff
@@ -0,0 +1,109 @@
+name: velocypack-test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CMAKE_BUILD_PARALLEL_LEVEL: 4
+  CXX_STANDARD: 20
+  SANITIZER: false
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          # GCC builds
+          - {os: ubuntu-latest, compiler: gcc, version:  '9',
+             build_type: Release, hash_type: xxhash }
+          - {os: ubuntu-latest, compiler: gcc, version: '10',
+             build_type: Release, hash_type: fasthash }
+          - {os: ubuntu-latest, compiler: gcc, version: '11',
+             build_type: Release, hash_type: xxhash }
+          - {os: ubuntu-latest, compiler: gcc, version: '10',
+             build_type: Debug, hash_type: xxhash,
+             flags: '-O0 -fno-inline --coverage', coverage: true}
+          - {os: ubuntu-latest, compiler: gcc, version: '10',
+             build_type: Debug, hash_type: xxhash,
+             flags: '-fsanitize=address -fsanitize=leak -fsanitize=undefined -fsanitize-address-use-after-scope -fno-omit-frame-pointer',
+             ld_flags: '-fsanitize=address -fsanitize=leak -fsanitize=undefined -fsanitize=address-use-after-scope -fno-omit-frame-pointer'}
```

Comment:
It would be good to have some variable set here in case we are building with sanitizers.
Right now the script `scripts/travis-test.sh` filters on whether the `SANITIZER` variable is set to `true` (currently never), and if it were, it would not run the fuzz tests.
I imagine that if we build with sanitizers, we probably want to reduce the amount of fuzz testing to keep the execution times at bay. If execution of the fuzz tests with sanitizers is fast enough, then we don't need to set a variable here, and we can remove the `if` condition from `scripts/travis-test.sh` that check if sanitizers are enabled.

Reply:
Should now set `SANITZER` environment variable, though I do think that the fuzz tests did run on the sanitizer run and there wasn't too much of an issue